packages/sdk/src/utils/chain-constants.ts

```diff
@@ -194,6 +196,38 @@ export const CONTRACT_ADDRESSES: {
     },
     l2: DEFAULT_L2_CONTRACT_ADDRESSES,
   },
+  [L2ChainID.ZORA_GOERLI]: {
```

Comment:
// Zora Goerli
```suggestion
  [L2ChainID.ZORA_GOERLI]: {
```
Adding a commit to this pr to force checks to run correctly

Reply:
Hmm looks like I may have to fork and merge I'll link to this pr